index.js

```diff
@@ -36,10 +36,10 @@ function CachedPersistence (opts) {
   })
 
   this._onMessage = function onSubMessage (packet, cb) {
-    var decoded = JSON.parse(packet.payload)
-    var clientId = decoded.clientId
-    for (var i = 0; i < decoded.subs.length; i++) {
-      var sub = decoded.subs[i]
+    const decoded = JSON.parse(packet.payload)
+    const clientId = decoded.clientId
+    for (let i = 0; i < decoded.subs.length; i++) {
```

Comment:
keep using var here, using let is still slower on node 12

Reply:
@mcollina Standard made this fix automatically, I should switch to let and add an ignore?

Reply:
switch to var and add an ignore

Reply:
Done

Reply:
Is it slower on specific version of node 12 as I just tested this and it seems that let (at least for me) is always faster with the exception of node 13 :thinking:
Could it be an operating system thing, I am running on `Ubuntu 20.10 (groovy)`?

```js
// speed.js
console.info(process.version);

let c;
console.time('let once');
for (c = 0; c < 1000000000; c++) {}
console.timeEnd('let once')

console.time('let');
for (let b = 0; b < 1000000000; b++) {}
console.timeEnd('let')

console.time('var');
for (var a = 0; a < 1000000000; a++) {}
console.timeEnd('var')
```

Node 12 var seems to be 1.5 times slower:
```
[~] $ node speed.js
v12.20.1
let once: (398.406ms, 399.045ms, 400.193ms)
let: (398.156ms, 399.415ms, 399.582ms)
var: (594.969ms, 623.737ms, 637.006ms)
```

Node 13 seems to have made let slower:
```
[~] $ node speed.js
v13.14.0
let once: (597.033ms, 620.504ms, 657.895ms)
let: (639.631ms, 642.129ms, 651.47ms)
var: (613.977ms, 618.612ms, 642.168ms)
```

Node 14 seems to be uniformly slower:
```
[~] $ node speed.js
v14.15.4
let once: (397.957ms, 398.932ms, 399.552ms)
let: (581.774ms, 607.696ms, 612.382ms)
var: (588.254ms, 670.176ms, 616.371ms)
```

Node 15 seems to make them both ~400ms:
```
[~] $ node speed.js
v15.8.0
let once: (536.674ms, 629.69ms, 651.413ms)
let: (397.924ms, 398.234ms, 398.428ms)
var: (398.325ms, 398.385ms, 399.639ms)
```